modules/openy_features/openy_media/openy_media.module

```diff
@@ -63,3 +64,14 @@ function openy_media_preprocess_entity_embed_container(array &$variables) {
     '_' => '-',
   ]);
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * Please see https://www.drupal.org/node/2749695#comment-11950133.
+ */
+function openy_media_media_presave(MediaInterface $media) {
+  if ('image' == $media->bundle() && !$media->isNew()) {
```

Comment:
This should be places in `openy_media_image` module.

Reply:
According to standards conditions should be reversed `$media->bundle() == 'image'`.

Reply:
Done.